create/cloudvm.go

```diff
@@ -0,0 +1,137 @@
+	Name                string            `arg:"" default:"" help:"Name of the CloudVM instance. A random name is generated if omitted."`
+	Location            string            `default:"nine-es34" help:"Location where the CloudVM instance is created."`
+	MachineType         string            `default:"nine-standard-1" help:"MachineType defines the sizing for a particular cloud vm."`
+	Hostname            string            `default:"" help:"Hostname allows to set the hostname explicitly. If unset, the name of the resource will be used as the hostname. This does not affect the DNS name."`
+	PowerState          string            `default:"on" help:"PowerState specifies the power state of the cloud VM. A value of On turns the VM on, shutdown sends an ACPI signal to the VM to perform a clean shutdown and off forces the power off immediately."`
+	OS                  string            `default:"ubuntu22.04" help:"OS which should be used to boot the VM."`
+	BootDisk            map[string]string `default:"{name:\"root\",size:\"20Gi\"}" help:"BootDisk that will be used to boot the VM from. Needs to be in the following format: {name:\"<name>\",size:\"<size>Gi\"}"`
+	Disks               map[string]string `default:"{}" help:"Disks specifies which additional disks to mount to the machine."`
+	PublicKeys          []string          `default:"[]" help:"PublicKeys specifies the SSH Public Keys that can be used to connect to the VM as root. The keys are expected to be in SSH format as defined in RFC4253. Immutable after creation."`
+	PublicKeysFromFile  string            `short:"cloud-config-f" predictor:"file" help:"CloudConfig via file. Has precedents over args. PublicKeys specifies the SSH Public Keys that can be used to connect to the VM as root. The keys are expected to be in SSH format as defined in RFC4253. Immutable after creation."`
+	CloudConfig         string            `default:"" help:"CloudConfig allows to pass custom cloud config data (https://cloudinit.readthedocs.io/en/latest/topics/format.html#cloud-config-data) to the cloud VM. If a CloudConfig is passed, the PublicKey parameter is ignored. Immutable after creation."`
+	CloudConfigFromFile string            `short:"cloud-config-f" predictor:"file" help:"CloudConfig via file. Has precedents over args. CloudConfig allows to pass custom cloud config data (https://cloudinit.readthedocs.io/en/latest/topics/format.html#cloud-config-data) to the cloud VM. If a CloudConfig is passed, the PublicKey parameter is ignored. Immutable after creation."`
+	Wait                bool              `default:"true" help:"Wait until CloudVM is created."`
+	WaitTimeout         time.Duration     `default:"600s" help:"Duration to wait for CloudVM getting ready. Only relevant if --wait is set."`
+}
+
+func (cmd *cloudVMCmd) Run(ctx context.Context, client *api.Client) error {
+	cloudVM, err := cmd.newCloudVM(client.Project)
+	if err != nil {
+		return err
+	}
+
+	c := newCreator(client, cloudVM, "cloudvirtualmachine")
```

Comment:
```suggestion
	c := newCreator(client, cloudVM, strings.ToLower(infrastructure.CloudVirtualMachineKind))
```

Reply:
added

Reply:
No, don't revert it to `"cloudvirtualmachine"` :smile: I meant just using `infrastructure.CloudVirtualMachineKind` without a `strings.ToLower`

Reply:
ah crap 😆